### PR TITLE
Don't consider RST comments as directives

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

**Comment by OskarStark** [comment]
Thank you 👍

File: tests/Rule/BlankLineAfterDirectiveTest.php [hunk 1]
```diff
@@ -83,5 +83,23 @@ public function checkProvider(): \Generator
...
+            ]),
+        ];
+
+        # see https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#comments
+        yield [
+            null,
+            new RstSample([
+                '..',
+                '   This is also a RST comment',
+            ]),
```
Comment (OskarStark) [inline tests/Rule/BlankLineAfterDirectiveTest.php:102]:
```suggestion
            ], 1),
```

**Comment by javiereguiluz** [comment]
Can I do something to help merge this? If this is not relevant anymore, let's close it. Thanks

**Comment by OskarStark** [comment]
Lets merge it

**Pull request merged**